kolibri/plugins/facility_management/assets/src/views/ClassEnrollForm.vue

```diff
@@ -21,14 +21,16 @@
     />
 
     <nav>
-      <span>
+      <span dir="auto" class="pagination-label">
         {{ $tr('pagination', { visibleStartRange, visibleEndRange, numFilteredUsers }) }}
       </span>
       <UiIconButton
         type="primary"
         :ariaLabel="$tr('previousResults')"
         :disabled="pageNum === 1"
         size="small"
+        :dir="isRtl ? 'rtl' : 'ltr'"
```

Comment:
what does adding a `dir` attribute to the icon button do?

Reply:
I added it because I was using attr selectors in the CSS a la `.class[dir='rtl']` - however, I was using hot-reloading so I was definitely seeing something different than what would be expected and - now that I'm not hot-reloading, I see the dir gets added automatically anyway.

Reply:
This allows me to target the styles using an attr selector a la `.pagination-button[dir='rtl'] {}`